pkgs/development/python-modules/gasp/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder,
+fetchgit, tkinter, pygame, python3, xvfb-run }:
+
+buildPythonPackage rec {
+  pname = "gasp";
+  version = "0.6.1";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-3h90H+E0pvDtpT/eQUsLjnnFholbxxk7K66exB6W3vo=";
+  };
+
+  propagatedBuildInputs = [ tkinter pygame ];
+  checkInputs = [ xvfb-run tkinter pygame ];
```

```suggestion
  checkInputs = [ xvfb-run ];
```
propagatedBuildInputs are still available here.